core/src/main/java/org/bitcoinj/wallet/Wallet.java

```diff
@@ -4201,8 +4197,8 @@ public void completeTx(SendRequest req) throws InsufficientMoneyException {
             checkArgument(!req.completed, "Given SendRequest has already been completed.");
             // Calculate the amount of value we need to import.
             Coin value = Coin.ZERO;
-            boolean requiresInputs = req.tx.getType() != Transaction.Type.TRANSACTION_SUBTX_RESETKEY;
-            boolean requiresNoFee = req.tx.getType() == Transaction.Type.TRANSACTION_SUBTX_RESETKEY;
+            boolean requiresInputs = true;
+            boolean requiresNoFee = false;
```

Comment:
Because this variable is always `false` code at lines 4243 and 4244 are never executed. If this code isn't needed, I suggest removing these two lines and the variable itself.

Reply:
See above comment